src/lib/linux-api/src/errno.rs

```diff
@@ -234,6 +234,7 @@ impl Errno {
     pub const ENOTRECOVERABLE: Self = Self::from_u32_const(bindings::LINUX_ENOTRECOVERABLE);
     pub const ERFKILL: Self = Self::from_u32_const(bindings::LINUX_ERFKILL);
     pub const EHWPOISON: Self = Self::from_u32_const(bindings::LINUX_EHWPOISON);
+    pub const EINTR: Self = Self::from_u32_const(bindings::LINUX_EINTR);
```

Comment:
Huh; how'd I miss that one? Could you add it to the `Debug` implementation match statement above too?